### PR TITLE
how-to: add doc on how to scale with TiUP

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

Add doc on how to scale with TiUP.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-4.0**, **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/2645
- Other reference link(s):

### CODE REVIEW

**Comment by ran-huang** [comment]
@wyuchen007 PTAL. Thanks~

File: how-to/scale/with-tiup.md [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+title: Scale the TiDB Cluster Using TiUP
+summary:
+Category: how-to
+---
+
+# Scale the TiDB Cluster Using TiUP
+
+The capacity of a TiDB cluster can be increased or decreased without affecting the online services.
+
+This document describes how to scale the TiDB, TiKV, PD or TiFlash nodes using TiUP. If you have not installed TiUP, [install TiUP on the Control Machine](/how-to/upgrade/using-tiup.md#install-tiup-on-the-control-machine) and import the cluster into TiUP before you scale the TiDB cluster.
```
Comment (ran-huang) [inline how-to/scale/with-tiup.md:11]: This link points to a file in https://github.com/pingcap/docs/pull/2232 (open).

File: how-to/scale/with-tiup.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+
+Access the monitoring platform at <http://10.0.1.5:3200> using your browser to monitor the status of the cluster.
+
+## 4. Scale in a TiFlash node
+
+If you want to remove the TiFlash node from the `10.0.1.4` host, take the following steps.
+
+> **Note:**
+>
+> The offline process described in this section does not delete the data on the offline node. If you need to take the node online again, delete the data manually.
```
Comment (yikeke) [inline how-to/scale/with-tiup.md:201]:
```suggestion
> The scale-in process described in this section does not delete the data on the node that goes offline. If you need to bring the node back again, delete the data manually.
```

File: how-to/scale/with-tiup.md [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+
+### 4.1 Take the node offline
+
+Take down the node to be scaled in. See [Take a TiFlash node down](/reference/tiflash/maintain.md#take-a-tiflash-node-down) for details.
```
Comment (yikeke) [inline how-to/scale/with-tiup.md:205]:
```suggestion
To take offline the node to be scaled in, refer to [Take a TiFlash node down](/reference/tiflash/maintain.md#take-a-tiflash-node-down).
```

File: how-to/scale/with-tiup.md [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+
+### 4.2 Check the node status
+
+Check whether the node has been offline successfully using Grafana or pd-ctl (the offline process takes some time).
```
Comment (yikeke) [inline how-to/scale/with-tiup.md:209]:
```suggestion
The scale-in process takes some time.

You can use Grafana or pd-ctl to check whether the node has been successfully taken offline.
```

File: how-to/scale/with-tiup.md [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+
+### 4.3 Shutdown the TiFlash process
+
+After the `store` corresponding to TiFlash disappears, or the `state_name` becomes `Tombstone`, execute the following command to shutdown the TiFlash process:
```
Comment (yikeke) [inline how-to/scale/with-tiup.md:213]: shutdown -> stop

Comment (yikeke) [inline how-to/scale/with-tiup.md:213]:
```suggestion
After the `store` corresponding to TiFlash disappears, or the `state_name` becomes `Tombstone`, execute the following command to stop the TiFlash process:
```

File: how-to/scale/with-tiup.md [hunk 6]
```diff
@@ -0,0 +1,229 @@
...
+
+### 4.1 Take the node offline
+
+Take down the node to be scaled in. See [Take a TiFlash node down](/reference/tiflash/maintain.md#take-a-tiflash-node-down) for details.
+
+### 4.2 Check the node status
+
+Check whether the node has been offline successfully using Grafana or pd-ctl (the offline process takes some time).
+
+### 4.3 Shutdown the TiFlash process
```
Comment (yikeke) [inline how-to/scale/with-tiup.md:211]:
```suggestion
### 4.3 Stop the TiFlash process
```

**Comment by ran-huang** [comment]
@yikeke Comments addressed. PTAL, thanks~

**Comment by yikeke** [comment]
For the steps, suggest adjusting the headings to ordered lists.

**Comment by ran-huang** [comment]
@yikeke Updated the numbered list. PTAL~

File: how-to/scale/with-tiup.md [hunk 7]
```diff
@@ -0,0 +1,231 @@
+---
+title: Scale the TiDB Cluster Using TiUP
+summary:
+Category: how-to
```
Comment (yikeke) [inline how-to/scale/with-tiup.md:4]: summary: xxx
category: how-to
aliases: ['/docs/stable/how-to/scale/with-tiup/']

Comment (yikeke) [reply how-to/scale/with-tiup.md:4]: @ran-huang Please fix the meta info after the doc special week.

Comment (ran-huang) [reply how-to/scale/with-tiup.md:4]: OK

**Pull request merged**